Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -622,15 +623,26 @@ extension HTTPClient {
         private var _isCancelled: Bool = false
         private var _taskDelegate: HTTPClientTaskDelegate?
         private let lock = Lock()
+        private let makeOrGetFileIOThreadPool: () -> NIOThreadPool
 
-        init(eventLoop: EventLoop, logger: Logger) {
+        public var fileIOThreadPool: NIOThreadPool {
```

Comment:
I just realized that we actually don't need to make this public. @Lukasa any objections to me making it internal?

Reply:
None at all.

Reply:
Done in https://github.com/swift-server/async-http-client/pull/614/commits/c3c68b2a27cc55c7597ddd83025b71fec7404ffc